docs/source/couchdb_tutorial.rst

```diff
@@ -63,16 +59,16 @@ Enable CouchDB in Hyperledger Fabric
 
 CouchDB runs as a separate database process alongside the peer, therefore there
 are additional considerations in terms of setup, management, and operations.
-A docker image of `CouchDB <https://hub.docker.com/r/hyperledger/fabric-couchdb/>`__
+A Docker image of `CouchDB <https://hub.docker.com/r/hyperledger/fabric-couchdb/>`__
```

Comment:
```suggestion
A Docker image of `CouchDB <https://hub.docker.com/_/couchdb/>`__
```
We are no longer building or shipping our own CouchDB image.

Reply:
done.